### ISSUE TITLE
Quick adaption to Multi-Label Change Detection

### ISSUE THREAD

**Issue by Bili-Sakura** [blaz-r/BTC-change-detection#2]
Awesome work!

I found the existing framework is designed for binary change detection. I wonder it is feasible to quick adpatation to multi-label change detection (e.g. building damage assessment change detection in xBD dataset) via replacing the out projection layer.

**Comment by blaz-r** [comment]
Hi,

I haven't tested the multi-label scenario, but I think basic adaptation could be made by simply changing the config to different channel numbers here:
https://github.com/blaz-r/BTC-change-detection/blob/3dcbce3326bbe15cbc0b4fce183b40c94cfb5348/configs/exp/BTC-B.yaml#L55-L61
by adding `out_channels: "num of classes"`.

This would then, of course, need a change in loss. Regular cross-entropy is already supported ("2chce" in config), but additional losses can be added here:
https://github.com/blaz-r/BTC-change-detection/blob/3dcbce3326bbe15cbc0b4fce183b40c94cfb5348/models/finetune_framework.py#L47

I think the only two sections that would require a code change are the dataset and the predict step. Transforms divide the labels by 255 here:
https://github.com/blaz-r/BTC-change-detection/blob/3dcbce3326bbe15cbc0b4fce183b40c94cfb5348/data/transforms.py#L100-L101
This is probably not suitable for a multi-label scenario.
And this part https://github.com/blaz-r/BTC-change-detection/blob/3dcbce3326bbe15cbc0b4fce183b40c94cfb5348/models/finetune_framework.py#L134-L137
would probably need to be `x.shape[1] >= 2`

**Comment by Bili-Sakura** [comment]
Thanks a lot！
